filters/alphanumeric/filter.py

```diff
@@ -0,0 +1,21 @@
+from tasks.TaskTypes import TaskType
```

Comment:
Your build is failing:probably because of missing

 from interfaces.SentenceOperation import SentenceOperation